usaspending_api/awards/migrations/0047_auto_20181126_1528.py

```diff
@@ -0,0 +1,811 @@
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='asian_pacific_american_own',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='black_american_owned_busin',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='c1862_land_grant_college',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='c1890_land_grant_college',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='c1994_land_grant_college',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='c8a_program_participant',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='city_local_government',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='community_developed_corpor',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='community_development_corp',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='contracts',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='corporate_entity_not_tax_e',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='corporate_entity_tax_exemp',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='council_of_governments',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='county_local_government',
+            field=models.NullBooleanField(),
+        ),
+        migrations.AlterField(
+            model_name='transactionfpds',
+            name='detached_award_proc_unique',
+            field=models.TextField(null=True, unique=True),
+        ),
+        # Drop "_like" indexes since we're converting from text to integer.
+        migrations.RunPython(
+            drop_like_pattern_ops_index('awards', 'transactionfpds', 'detached_award_procurement_id')
```

Comment:
Yeah.  I only move forward.  There is no going back.  CHAAAARGE!!!!!!

No, but for serious, is an oversight.  Will correct.  Will probably also just drop back to SQL since we've fully embraced it at this point.